### ISSUE TITLE
Go scripts fail if they don't import fmt

### ISSUE THREAD

**Issue by jaller94** [windmill-labs/windmill#640]
1. Create a Go script
2. Insert the code below
3. Run it

```go
func main(x string) (interface{}, error) {
  return x, nil
}
```

## Expected
It runs successfully. It has no dependencies.

## Actual
It fails because the Go module `fmt` isn't imported.

```

job 01838a0e-3434-9a74-781f-7e31e3d0434b on worker dt-worker-cX894-jQmss


--- GO DEPENDENCIES SETUP---

go: creating new go.mod: module mymod
go: to add module requirements and sums:
	go mod tidy

--- GO CODE EXECUTION ---

# command-line-arguments
mymod/main.go:17:9: undefined: fmt
mymod/main.go:24:9: undefined: fmt
mymod/main.go:30:9: undefined: fmt
mymod/main.go:35:9: undefined: fmt
mymod/main.go:38:5: undefined: fmt
mymod/main.go:39:5: undefined: fmt
mymod/main.go:40:5: undefined: fmt

--- ERROR ---

Error during execution of the script
last 5 logs lines:
mymod/main.go:30:9: undefined: fmt
mymod/main.go:35:9: undefined: fmt
mymod/main.go:38:5: undefined: fmt
mymod/main.go:39:5: undefined: fmt
mymod/main.go:40:5: undefined: fmt
```

**Comment by rubenfiszel** [comment]
Good catch, thank you. Will come up with a fix shortly.

**Comment by rubenfiszel** [comment]
Solved by not wrapping the file anymore in: [eabd835](https://github.com/windmill-labs/windmill/commit/eabd83580758121149b629285d8f4cb228c9a7ea)
